config/locales/en.yml

```diff
@@ -1915,6 +1915,8 @@ en:
   label_feeds_access_key_type: "RSS"
   label_file_plural: "Files"
   label_filter_add: "Add filter"
+  label_filter: "Filter"
+  label_filter_by_text: "Filter by text"
```

Comment:
This is not yet used, is it? If it really isn't, I'd remove it to avoid the risk of ending up with an unused string in case the desired behaviour is implemented not at all/differently.